### PR TITLE
Dev refactor

### PR DESCRIPTION
**Pull request by brrt2**
The high number of modified lines is a bit misleading as it mainly consisted of moving classes from package to package and adding javadocs.

### CODE REVIEW

File: client/src/main/java/gui/building/FleetDropController.java [hunk 1]
```diff
@@ -68,14 +70,27 @@
...
+
+  /**
+      * Creates FleetDropController instance.
+   *
+   * @param client    - client instance
+   * @param reactor - reactor instance
+   */
 
-  public FleetDropController(Client client, MessageReactor reactor) {
+  public FleetDropController(Client client, MessageProcessor reactor) {
```
Comment (EmiliaCiastek) [inline client/src/main/java/gui/building/FleetDropController.java]: change "reactor" name

File: client/src/main/java/gui/building/FleetDropController.java [hunk 2]
```diff
@@ -68,14 +70,27 @@
...
     sea = new Sea();
     this.client = client;
     this.reactor = reactor;
```
Comment (EmiliaCiastek) [inline client/src/main/java/gui/building/FleetDropController.java]: as above

File: client/src/main/java/gui/building/FleetDropController.java [hunk 3]
```diff
@@ -170,40 +176,37 @@ private void sendFleetToServer() {
...
-          event.setDropCompleted(success);
-          event.consume();
-          buildShip.setOpacity(0.2);
-          if (buildShip.getHeight() / FIELD_SIZE > 1) {
-            port.setDisable(true);
-          }
+        event.setDropCompleted(success);
+        event.consume();
+        buildShip.setOpacity(0.2);
+        if (buildShip.getHeight() / FIELD_SIZE > 1) {
```
Comment (EmiliaCiastek) [inline client/src/main/java/gui/building/FleetDropController.java]: magic numbers

File: client/src/main/java/gui/playing/PlayBoardController.java [hunk 4]
```diff
@@ -35,18 +40,14 @@ private void populateSeaWithSeaFields() {
       for (int n = 0; n < 10; n++) {
```
Comment (EmiliaCiastek) [inline client/src/main/java/gui/playing/PlayBoardController.java]: magic number: 10 -> 'boardSize' maybe

File: client/src/main/java/model/PossiblePositions.java [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+ */
+class PossiblePositions {
+
+  private List<SeaField> possible;
+
+  public PossiblePositions() {
+    possible = new ArrayList<>();
+  }
+
+  public PossiblePositions findPositions(Mast mast, Sea sea) {
```
Comment (EmiliaCiastek) [inline client/src/main/java/model/PossiblePositions.java]: should return possible, not this

File: client/src/main/java/model/PossiblePositions.java [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+    if (mast.getColumn() <= bottomBoundary) {
+      possible.add(sea.getSeaFieldByPosition(Position.right(mast.position())));
+    }
+    if (mast.getColumn() >= upperBoundary) {
+      possible.add(sea.getSeaFieldByPosition(Position.left(mast.position())));
+    }
+    return this;
+  }
+
+  public void makePositionClickable() {
```
Comment (EmiliaCiastek) [inline client/src/main/java/model/PossiblePositions.java]: should accept 'possible' list -> we could move it to another class and test

File: client/src/main/java/connection/FleetSender.java [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+  private Client client;
+  private Player player;
+
+  public FleetSender(Client client, Player player) {
+    this.client = client;
+    this.player = player;
+  }
+
+  public void sendFleetToServer() {
```
Comment (igasm) [inline client/src/main/java/connection/FleetSender.java]: so many things is happening here...

File: client/src/main/java/gui/building/FleetDropController.java (hunk 3 above)
Comment (patiem) [reply client/src/main/java/gui/building/FleetDropController.java]: this line is number of masts in ship

File: client/src/main/java/model/PossiblePositions.java (hunk 6 above)
Comment (patiem) [reply client/src/main/java/model/PossiblePositions.java]: Yes, it is just beginning of refactoring

**Pull request merged**